gerrymandering/gerrymandering.ipynb

```diff
@@ -97,9 +102,51 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = dd.io.parquet.read_parquet('./data/census.snappy.parq', engine='pyarrow')\n",
-    "df = df.persist()\n",
-    "census_points = hv.Points(df, kdims=['easting', 'northing'], vdims=['race'])"
+    "df = dd.read_parquet('./data/census.snappy.parq', engine='fastparquet')\n",
+    "# Load just a fraction of the points if your computer doesn't have enough memory\n",
+    "# df = df.sample(frac=0.01)\n",
+    "df = df.persist()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# census_points = hv.Points(df, kdims=['easting', 'northing'], vdims=['race'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shaded = df.hvplot.points(\n",
+    "    'easting', 'northing',\n",
+    "    datashade=True,\n",
+    "    # rasterize=True,\n",
```

Comment:
Is there a reason not to use `rasterize=True`? See e.g. the second to last plot in census.ipynb for an example of that.

Reply:
This is what I get when I swap `datashade=True` for `rasterize=True`. No legend and some or all the options are not propagated (e.g. width).

![image](https://github.com/holoviz-topics/examples/assets/35924738/db0bba1c-a8df-464f-ba2b-bce5cb960e62)

Repr of this object:
```
:DynamicMap   []
   :Overlay
      .Tiles.I      :Tiles   [x,y]
      .ImageStack.I :ImageStack   [easting,northing]   (a,b,h,o,w)
```